Program.cs

```diff
@@ -153,32 +178,34 @@ static void Main(string[] args)
                 "finding the axe and cutting down the tree.\nUse w, s, a and d to move around and x to exit the game.  Enjoy!\n");
 
             // loop for continual movement
-            while (JayW.getHealth() > 0)
+            while (true)
             {
-                Console.WriteLine("Where do you want to go traveler? The X represents where you are now.\n");
-                // printing out game board
-                PrintGameBoard(tileGrid, currRow, currCol);
+                if (canRun)
```

Comment:
you can do if(canRun && JayW.getHealth() >0) instead

Reply:
That's a good point. I will make that change